src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import Mapping
+
+from sentry.sentry_metrics.configuration import UseCaseKey
+
+
+class UseCaseID(Enum):
+    TRANSACTIONS = "transactions"
+    SESSIONS = "sessions"
+
+
+# UseCaseKey will be renamed to MetricPathKey
+METRIC_PATH_MAPPING: Mapping[UseCaseID, UseCaseKey] = {
+    UseCaseID.SESSIONS: UseCaseKey.RELEASE_HEALTH,
+    UseCaseID.TRANSACTIONS: UseCaseKey.PERFORMANCE,
+}
+
+
```

Comment:
Have we considered adding [TABLE_MAPPING](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/postgres/models.py#L88-L91) to here as well?